osprey/processes/wps_convert.py

```diff
@@ -122,10 +122,7 @@ def _handler(self, request, response):
             process_step="process",
         )
 
-        try:
-            convert(config_file)
-        except RecursionError:
-            close_logger()
+        convert(config_file)
```

Comment:
Much better having this handled in the actual package. You can remove the import for `close_logger` if we are no longer using it.